### PR TITLE
Frontier - add assetlist Oraichain mainnet

### PR DESCRIPTION
**Pull request by ducphamle2**

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -1245,6 +1245,36 @@
...
+      "name": "Oraichain",
+      "display": "orai",
+      "symbol": "ORAI",
+      "ibc": {
+        "source_channel": "channel-13",
+        "dst_channel": "channel-216",
+        "source_denom": "orai"
+      },
+      "logo_URIs": {
+        "png": "https://orai.io/images/logos/logomark-dark.png",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:1274]: please upload the images to the \images\ folder and name them 'orai.png'

Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:1274]: please upload the images to the \images\ folder and name them 'orai.png'

**Comment by JeremyParish69** [comment]
Please add the the frontier file instead, not the main file.

**Comment by JeremyParish69** [comment]
Please add the the frontier file instead, not the main file.

**Comment by JeremyParish69** [comment]
Please update the logo URI's to 'https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/orai.png' and '... .svg'.

**Comment by JeremyParish69** [comment]
Please update the logo URI's to 'https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/orai.png' and '... .svg'.

**Comment by JeremyParish69** [comment]
Pending liquidity
Pending addition to Frontier

**Comment by ducphamle2** [comment]
> Pending liquidity Pending addition to Frontier

are these the actions that I need to take to finish this PR?

**Comment by JeremyParish69** [comment]
> > Pending liquidity Pending addition to Frontier
> 
> are these the actions that I need to take to finish this PR?

Yes, please add liquidity onto Osmosis.
You can use a staging link if you create a PR for adding the token onto Osmosis Frontier.
https://docs.osmosis.zone/integrate/frontend.html#how-to-add-an-asset-onto-the-osmosis-assets-page

**Pull request closed without merging**